### ISSUE TITLE
parse error

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#546]
```
What's the problem?

cribeFire couldn't publish your post. Here's the error message that bubbled up: 
parse error. not well formed



What browser are you using?

Firefox



What version of ScribeFire are you running?
4.0
```

-----
Original issue reported on code.google.com by `Acudoc2...@gmail.com` on 14 Jan 2012 at 5:44

**Comment by GoogleCodeExporter** [comment]
```
what blog type are you using?
```

-----
Original comment by `scribefi...@gmail.com` on 8 Apr 2014 at 3:04

**Comment by GoogleCodeExporter** [comment]
```
Fixed for blogger blogspot in version 4.3.1
```

-----
Original comment by `scribefi...@gmail.com` on 14 Apr 2014 at 7:08
* Changed state: **Fixed**
